workers/check/data_item.py

```diff
@@ -44,7 +46,11 @@ def callback(client_state, channel, method, properties, input_message):
             )
 
             # perform actual action with items
-            processor.do_work(cursor.fetchall())
+            processor.do_work(
+                cursor.fetchall(),
+                do_field_level_checks=is_step_required(settings.FIELD_COVERAGE_STEP),
```

Comment:
This one should run if either `field_coverage` or `field_quality`. We'd need to add the guards in `field_level_checks`.